### PR TITLE
main-1.18 compatibility with Create 0.5.1a+

### PR DESCRIPTION
**Pull request by nanonestor**
Update main-1.18 to be compatible with Create 0.5.1a and newer

### CODE REVIEW

**Comment by nanonestor** [comment]
^  As requested I closed the previous PR and made a new one to commit to main-1.18 branch

**Comment by MuradAkh** [comment]
Thanks for the PR! will get it reviewed, merged, and release when I get a chance

File: gradle.properties [hunk 1]
```diff
@@ -4,7 +4,7 @@ org.gradle.jvmargs  = -Xmx3G
 org.gradle.daemon   = false
 
 # Core properties
-mod_version         = 1.2.0dev
+mod_version         = 1.2.7
```
Comment (muradakh) [inline gradle.properties:7]: You don't need to change this, the build will change it automatically

Comment (nanonestor) [reply gradle.properties:7]: Oof ok.  😭 Is this something to make another PR for?

Comment (muradakh) [reply gradle.properties:7]: you can just push a new commit to your branch and it should show up

**Comment by MuradAkh** [comment]
Is the build running for you locally? The CI build seems to be failing

**Comment by nanonestor** [comment]
Well uh sort of - i got the clone I have working *mostly*.  For some reason textures are breaking for all items and half or so of blocks.  I can't figure out why.  

Using IntelliJ IDEA community edition. Simply importing a fresh copy of main-1.18 branch and importing build.gradle will build but reproduce the same texture problems so idk what is wrong. :(

**Pull request closed without merging**